policy/modules/system/fwupd.te

```diff
@@ -0,0 +1,176 @@
+policy_module(fwupd)
+
+########################################
+#
+# Policy for firmwate update daemon and utility
+# Debian package fwupd
+#
+
+########################################
+#
+# Declarations
+#
+
+attribute_role fwupd_roles;
+type fwupd_t;
+type fwupd_exec_t;
+init_daemon_domain(fwupd_t, fwupd_exec_t)
```

Comment:
Why does the command line tool run in this domain if it is a daemon? Which part (daemon/cli) does the firmware install?